.circleci/config.yml

```diff
@@ -1,145 +1,287 @@
-
-mkdir-tests-dir: &mkdir-tests-dir
-  run: mkdir -p "${TESTS_DIR}"
-
-store-test-results: &store-test-results
-  type: test-results-store
-  path: /tmp/test-results
-
-store-artifacts: &store-artifacts
-  type: artifacts-store
-  path: /tmp/test-results
-  destination: test-results
-
-version: 2
-jobs:
-  verify:
-    <<: *working_dir
+# This file was generated by the excavator check 'excavator/manage-circleci' as specified in .circleci/template.sh.
+# To request a modification to the general template, file an issue on Excavator.
+# To manually manage the CircleCI configuration for this project, remove the .circleci/template.sh file.
+
+version: 2.1
+
+orbs:
+  go: palantir/go@0.0.29
+  godel: palantir/godel@0.0.29
+
+homepath: &homepath
+  homepath: /home/circleci
+
+gopath: &gopath
+  gopath: /home/circleci/go
+
+working_directory: &working_directory
+  working_directory: /home/circleci/go/src/github.com/palantir/conjure-go
+
+executors:
+  circleci-go:
     docker:
-      - image: palantirtechnologies/go:go-darwin-linux-no-cgo-1.15.2-java-11-t60
-        environment:
-          CGO_ENABLED: 0
-    steps:
-      - checkout
-      - *go-version
-      - *godel-cache-restore
-      - *godel-version
-      - *godel-cache-save
-      - run: ./godelw verify --apply=false --skip-test
-  test:
-    <<: *darwin-linux-no-cgo
-    steps:
-      - checkout
-      - *go-version
-      - *godel-cache-restore
-      - *godel-version
-      - *godel-cache-save
-      - *define-tests-dir
-      - *mkdir-tests-dir
-      - run: ./godelw test --tags=none --junit-output="$TESTS_DIR/$CIRCLE_PROJECT_REPONAME-tests.xml" -- -p=1
-      - *store-test-results
-      - *store-artifacts
+      - image: cimg/go:1.16-browsers
+    <<: *working_directory
+
+all-tags-filter: &all-tags-filter
+  filters:
+    tags:
+      only: /.*/
+
+jobs:
+  # rendered version of the "test" job -- customized because it includes the "palantirtechnologies/conjure-verification-server:0.18.5"
+  # Docker image and uses the "--tags=conjure-verifier" flag for "./godelw test".
   conjure-verifier:
-    <<: *working_dir
+    <<: *working_directory
     docker:
-      - image: palantirtechnologies/go:go-darwin-linux-no-cgo-1.15.2-t60
-        environment:
-          CGO_ENABLED: 0
+      - image: cimg/go:1.16-browsers
       - image: palantirtechnologies/conjure-verification-server:0.18.5
     steps:
       - checkout
-      - *go-version
-      - *godel-cache-restore
-      - *godel-version
-      - *godel-cache-save
-      - *define-tests-dir
-      - *mkdir-tests-dir
-      - run: ./godelw test --tags=conjure-verifier --junit-output="$TESTS_DIR/$CIRCLE_PROJECT_REPONAME-tests.xml"
-      - *store-test-results
-      - *store-artifacts
-  dist:
-    <<: *darwin-linux-no-cgo
-    steps:
-      - checkout
-      - *go-version
-      - *godel-cache-restore
-      - *godel-version
-      - *godel-cache-save
-      - run: ./godelw dist
+      - run:
```

Comment:
This section is mostly lifted from the orb. This job needs to be custom because the executor has multiple docker images (includes `palantirtechnologies/conjure-verification-server:0.18.5` for tests). Otherwise, this could just the `godel/tests` job with the tag value of `conjure-verifier`.

I'm not sure that it's possible to manually define an executor that uses multiple Docker images -- if it is, then it's possible that we could leverage that instead. The other approach would be to define an orb action that basically fills in all the pre/post steps, and then we could just use that and drop in the custom logic for running tests and setting up the executor.

Reply:
Actually maybe that is possible... let me try this out.

Reply:
Huh, that actually worked -- hooray for rubber-ducking!